### ISSUE TITLE
Confirmed Technics Ottava SC-C70

### ISSUE THREAD

**Issue by fritzi-blizi** [milaq/YCast#122]
Technics Radio Technics Ottava SC-C70 is running too..
technics.vtuner.com

**Comment by lars721** [comment]
Hi fritzi-blizi,
this sounds good that YCast works with your Technics Ottava. I will also try with my Technics ST-C700D.
Seems that both had similar Firmware changes in the relation to vTuner issues in the past.
Hopefully they have similar firmware for vTuner included.
I am not sure about the correct vTuner URL because Wireshark shows me as last network commands when trying to reach the Server for using "Internet Radio" a ping to IP 75.126.133.179 with no answer.
Destination of that IP is in US - a service provider of vTuner company.
I will try to build up my own DNS server to reconnect my player towards YCast server. But I am not familar with this.

And be careful to **not** install the new firmware released by Technics on January, 10 th 2023 !!
It will delete the complete Internet Radio functionality of your Technics Network Player !!!!

Lars

**Comment by fritzi-blizi** [comment]
Hi Lars,my YCAST server runs great with my Technics SC-C70. The site is technics.vtuner.com . Thanks for the software!The receiver looks for a second address in Japan for new firmware. I blocked this query with Pi-Hole on the Raspberry server. Now i have no longer a update information on the display..A site in the Nederland describes, how yo can built up a complete server with Pi-Hole and so on. http://alriklubbers.com/2020/02/22/install-vtuner-alternative-alongside-pi-hole/ It was for me a little bit easier and complete…It is a big mess, that Technics destroys a function of the receiver with a software update!Best regardsHans-HermannVon meinem iPad gesendetAm 12.01.2023 um 20:38 schrieb lars721 ***@***.***>:﻿
Hi fritzi-blizi,
this sounds good that YCast works with your Technics Ottava. I will also try with my Technics ST-C700D.
Seems that both had similar Firmware changes in the relation to vTuner issues in the past.
Hopefully they have similar firmware for vTuner included.
I am not sure about the correct vTuner URL because Wireshark shows me as last network commands when trying to reach the Server for using "Internet Radio" a ping to IP 75.126.133.179 with no answer.
Destination of that IP is in US - a service provider of vTuner company.
I will try to build up my own DNS server to reconnect my player towards YCast server. But I am not familar with this.
And be careful to not install the new firmware released by Technics on January, 10 th 2023 !!
It will delete the complete Internet Radio functionality of your Technics Network Player !!!!
Lars

**Comment by lars721** [comment]
Yes! Successfully installed Internet Radio for my Technics **ST-C700D** too.

I used the pdf file instruction found here in AVForum https://www.avforums.com/threads/vtuner-internet-radio-stop-working-marantz-denon-3-case.2269858/page-8#post-29660484

- I used my already installed raspberry pi. It had already a static IP address fixed in my Router. No need to change any DHCP settings on the raspberry. (My Technics has connected network wire and no Wifi !)
-   installed dnsmasq and dnsutils
- add entry: address=/vtuner.com/192.168... (local IP address of my Raspberry) in file dnsmasq.conf
- installed YCast
- Changed network settings on my Technics player: DHCP off, fixed IP address, Gateway IP of my router, **first** DNS server (IP address of my RaspberryPi) (the pdf instruction pointed to change the secondary DNS server address, but this didn't worked for my Technics
I assume that YCast should work with all following Technics models: SU-G30, ST-C700D, SC-C500, SU-C550, SC-C70
